### PR TITLE
Add a shared utils folder for shared functions and more functionality to the question list

### PR DESCRIPTION
**Pull request by ESC8504**
Created a shared utils folder for shared functions, added sort by helpfulness function, and reformat date function.
Added more details to the Questionlist.jsx file and sorted questions in order of helpfulness

### CODE REVIEW

**Review by tomki1 — CHANGES_REQUESTED** [review]
It looks good! Maybe add at least one test to test sort by helpfulness function?

**Comment by ESC8504** [comment]
Forgot about the test, will do it!

**Review by tomki1 — APPROVED** [review]
Looks good!

File: client/utils/reFormatDate.js
Comment (LLaxt) [inline client/utils/reFormatDate.js:1]: looks good! I wonder if there is an in-built way to format date?
I'd also change the filename to "reformatDate" just for consistency, but not important!

**Review by LLaxt — APPROVED** [review]

**Pull request merged**